ModerationPolicy.md

```diff
@@ -0,0 +1,355 @@
+-   Via a new Post in the same thread as the Post being requested for Moderation,
+-   Via a new Post in the private expressjs/moderation repository.
+
+Note that Collaborators may Moderate non-Collaborator Posts at any time without
+submitting an initial request (see: [Non-Collaborator Posts][]).
+
+Use of the [report@expressjs.com][] email address -- or private email to individual
+Moderation Team members -- is appropriate when the individual requesting the
+Moderation does not feel comfortable directly or publicly making the request.
+All emails sent to the [report@expressjs.com][] address are currently forwarded
+to all members of the Moderation Team.
+
+When a request is sent by email to the [report@expressjs.com][] (or directly to a
+Moderation Team member) the moderation team must log the issue internally and
+report it periodically to the TC.
+
+Requests should contain as much information and context as possible, including
+the URL and a screenshot of the Post in question. Screenshots may be modified
+to obscure obscene or offensive content.
+
+External public venues or social media services such as Twitter must never be
+used to request Moderation.
+
+Collaborators must never discuss the specific details of a Moderation request
+in any public forum or any social media service outside of the expressjs GitHub
+Organizations.
+
+Note that quoting the original content of a Post within a Moderation request or
+expressjs/moderation repository issue is not a violation of the
+[Code of Conduct][]. However, discretion is advised when including such quotes
+in requests posted to public repositories.
+
+Requests for Moderation that do not appear to have been submitted in good faith
+with intent to address a legitimate [Code of Conduct][] violation will be
+ignored.
+
+## Consideration of Intent
+
+Before Moderating a Post, Collaborators should carefully consider the possible
+intent of the author. It may be that the author has simply made an error or is
+not yet familiar with the [Code of Conduct][]; or it may be that cultural
+differences exist, or that the author is unaware that certain content is
+inappropriate. In such cases, the author should be given an
+opportunity to correct any error that may have been made.
+
+Note, however, that unfamiliarity with the [Code of Conduct][] does not excuse
+a Post from Moderation.
+
+## Guidelines and Requirements
+
+-   All Posts are expected to respect the expressjs [Code of Conduct][].
+-   Any Collaborator with triage permission to a given repository (except
+    expressjs/moderation) may Moderate Posts within that repository's issue tracker.
+    Only the Moderation team is allowed to moderate posts on expressjs/moderation.
+-   The Moderation Team serves as the final arbiter for all Moderation issues.
+-   Moderation Team members may Remove or Block an individual from the expressjs
+    GitHub Organizations.
+-   For any Removal or Blocking action, an issue describing the reasons for the
```

Comment:
```suggestion
-   For any Removal or Blocking actions, an issue describing the reasons for the
```

Reply:
I think "action" here refers to each instance of removal or blocking individually, rather than multiple actions collectively.